### PR TITLE
initial terminology and text review

### PR DESCRIPTION
**Pull request by henriterhofte**
### Checklist

- [ ] Tests have been written/updated (if necessary)
  - [ ] All tests are passing
- [ ] Docs have been updated (if necessary)
- [ ] Updated `In Progress` issue to `In Review`
- [ ] A team member who has not worked on this branch has reviewed this PR
- [* ] Commits are properly formatted and messages are descriptive
- [ ] The PR title and description are clear and concise

### Description

This PR contains mu suggestions for textual and terminology improvements: to keep the text focuses at the end users (occupants) and not the energy doctors, to refrain from using technical design terminology such as DateSouce, EnergyQuery and EnergyDoctor, which are not intended for consumption by end users, to shorten texts where possible and to be more in line with other terminology and spelling used in the rest of the app.

### Linked Issues

There is a related PR for the https://github.com/energietransitie/needforheat-manuals repo. 

### Additional Context

I could not check the changes locally. However, it should only include textual changes. Unlwee you're confident that all is well, I may be wise to review and merge the PR only after Winnovation, so as not to endanger the Winnovation demo?a

### CODE REVIEW

File: apps/expo/src/lang/en-US.json [hunk 1]
```diff
@@ -234,52 +234,52 @@
...
           }
         },
         "weather_location_result_screen": {
-          "title": "Check Results",
-          "subtitle": "Is your home inside this weather zone?",
-          "send_button": "Send my query!",
-          "back_button": "Try again",
+          "title": "Weather zone to send",
+          "subtitle": "Is your home located in this weather zone?",
+          "send_button": "Yes (send this zone)",
```
Comment (Labhatorian) [inline apps/expo/src/lang/en-US.json:257]: This is a bit too long for the button. Maybe something like `Yes (send zone)`?
If we can lose 2 characters it'll fit on all screens.

**Review by Labhatorian — CHANGES_REQUESTED** [review]
We have no issues with these translations, just 2 small things

File: apps/expo/src/lang/nl-NL.json [hunk 2]
```diff
@@ -29,16 +29,16 @@
...
         "super_late": "Offline",
         "next_update": "Volg. in",
         "hours": "u"
       },
       "explanation_sheet": {
-        "title": "Welkom bij de NeedForHeat App!",
-        "description": "Dit scherm toont een lijst van Databronnen die u moet voltooien.\n\nDit moet u weten:\n- Open Databronnen zijn wit.\n- Vergrendelde Databronnen zijn grijs.\n- Onboarded Databronnen zijn groen.\n\nUw doel is om alle Databronnen te voltooien en volledig deel te nemen aan de campagne! Uw voortgang wordt weergegeven in de progressiebalk bovenaan.\n\nTik op een open Databron om te beginnen. Gebruik het ?-pictogram voor veelgestelde vragen.\n\nU kunt deze uitleg opnieuw openen door op het groene ?-pictogram rechtsonder op het scherm te drukken wanneer deze uitleg verdwijnt."
+        "title": "Korte uitleg",
+        "description": "Dit scherm toont een lijst met datbronnen die geactiveerd moeten worden: installeer apparaten, verleen toegang tot online bronnen, en beantwoord vragen.\n\nGroene bronnen zijn geactiveerd, grijze worden beschikbaar nadat één of meer witte bronne zijn geactiveerd.\n\nSelecteer het groene ?-pictogram rechtsonder om deze uitleg opnieuw te bekijken."
```
[the remark this answers is not in the document]
Comment (henriterhofte) [reply apps/expo/src/lang/nl-NL.json:41]: fixed in new commit

File: apps/expo/src/lang/en-US.json (hunk 1 above)
Comment (henriterhofte) [reply apps/expo/src/lang/en-US.json:257]: resolved in new commit

**Comment by henriterhofte** [comment]
> We have no issues with these translations, just 2 small things

I fixed the issues. However, this reminds me that the explanation_sheet does not have a proper left and right white margin (text runs almost off the side of my smartphone screen).. can you fix this easily? Of so, please do.

**Review by Labhatorian — APPROVED** [review]
Added margins and all seems well

**Pull request merged**